InfiniTAM/ORUtils/Matrix.h

```diff
@@ -49,6 +49,12 @@ namespace ORUtils {
 		T m[s*s];
 	};
 
+    template<class T>
```

Comment:
Please can you fix the indentation of this?

Reply:
Will do.